SwiftyBluetooth/Source/CentralProxy.swift

```diff
@@ -200,6 +203,56 @@ extension CentralProxy {
         }
     }
     
+    func connect(peripheralUUID: UUID, serviceUUIDs: [CBUUID], timeout: TimeInterval, _ callback: @escaping ConnectPeripheralCallback) {
```

Comment:
Custom stuff, can be made in an extension instead, goal of library isn't to provide stuff like this.